### PR TITLE
Tests for mfsr.go

### PR DESCRIPTION
**Pull request by Zanadar**
Adds tests for `msfr.go`. Does not hit coverage target of 80%.
Remove code that was transforming errs when the version file didn't exist, which may have confused callers of the `Version` method.

### CODE REVIEW

File: repo/fsrepo/migrations/mfsr.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ func (rp RepoPath) Version() (int, error) {
 	}
 
 	fn := rp.VersionFile()
-	if _, err := os.Stat(fn); os.IsNotExist(err) {
-		return 0, VersionFileNotFound(rp)
+	if _, err := os.Stat(fn); err != nil {
```
Comment (whyrusleeping) [inline repo/fsrepo/migrations/mfsr.go]: why was this changed?

Comment (whyrusleeping) [inline repo/fsrepo/migrations/mfsr.go]: Ah, that makes sense then.

**Comment by whyrusleeping** [comment]
@Zanadar do you mind rebasing this PR on latest master? To get CI on jenkins running properly we need the new makefile changes

**Comment by Zanadar** [comment]
Rebased.
